velox/expression/ExprToSubfieldFilter.h

```diff
@@ -15,9 +15,16 @@
  */
 #pragma once
 
-#include "velox/type/Filter.h"
+#include <velox/core/Expressions.h>
+#include <velox/core/ITypedExpr.h>
+#include <velox/core/PlanFragment.h>
+#include <velox/core/PlanNode.h>
+#include <velox/type/Filter.h>
+#include <velox/type/Subfield.h>
 
-namespace facebook::velox::common::test {
+using namespace facebook::velox;
+
+namespace facebook::velox::exec {
```

Comment:
Looks like this should be velox::core namespace, same one as TypedExprPtr. It then makes sense to move this file to 'core' directory. Sorry,  I mistakenly thought that 'expression' directory would be the right one.

Reply:
It cannot live in velox::core. The issue is that it uses Expr and expression uses core. So leads to circular dependency (I tried). So to me seems exec is the right place.

Reply:
Got it. Thank you for explaining.

Reply:
Curious where does the usage of "Expr" is coming from.